examples/i18n/src/i18n.rs

```diff
@@ -0,0 +1,96 @@
+use fluent::{FluentArgs, FluentBundle, FluentResource};
+use strum_macros::EnumIter;
+use unic_langid::LanguageIdentifier;
+
+use std::borrow::Borrow;
+use std::str::FromStr;
```

Comment:
I would perhaps switch groups - i.e. imports from `std` would be first and external crates the second ; your choice

Reply:
Makes sense. Done.